hack/ostree_tag.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
+declare -a tags
 if pkg-config ostree-1 2> /dev/null ; then
-	echo containers_image_ostree
+	tags+=(containers_image_ostree)
 fi
+
+if test $(${GO:-go} env GOOS) != "linux" ; then
+	exit 0
+fi
+cc -E - $(pkg-config --cflags ostree-1) > /dev/null 2> /dev/null << EOF
+#include <ostree-1/ostree.h>
```

Comment:
This does not seem actually representative; `pkg-config --cflags ostree-1` returns something like
```
-I/usr/include/ostree-1 -I/usr/include/gio-unix-2.0/ -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -pthread 
```
and there is an `/usr/include/ostree-1/ostree.h` file. So, on a Fedora Linux, this check works whether or not `pkg-config` recognizes the package, and it _only_ works if `libostree` has headers in `/usr/include`, defeating the point of `pkg-config`.

Note that both CGo users use `#include <ostree.h>`.